addons/l10n_lv/data/account_tax_template_data.xml

```diff
@@ -0,0 +1,557 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+  <!-- Sales -->
+  <!-- 21% VAT, VAT report r41 + r51 + PVN 1.1. -->
+  <record id="account_tax_template_sales_21_vat41S" model="account.tax.template">
```

Comment:
```suggestion
  <record id="account_tax_template_sales_21_vat41S" model="account.tax.template">
    <field name="sequence">100</field>
```
Could we please add the `sequence` field to each tax, which is an increasing number? (e.g. 100, 110, 120 etc.)